assets/js/modules/ads/components/notifications/SetupSuccessSubtleNotification.stories.js

```diff
@@ -53,10 +53,16 @@ export default {
 	decorators: [
 		withQuery,
 		( Story, { parameters } ) => {
+			const setupRegistry = ( registry ) => {
+				if ( parameters?.setupRegistry ) {
+					parameters.setupRegistry( registry );
+				}
+			};
```

Comment:
Please remove where unused. If no `setupRegistry` is being used then this decorator is unnecessary and can be removed.